app/src/androidTest/java/org/mozilla/reference/browser/helpers/ext/String.kt

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.reference.browser.ext
+
+import android.net.Uri
+
+import java.net.URI
+import java.net.URISyntaxException
+
+// Extension functions for the String class
+
+
+/**
+ * If this string starts with the one or more of the given [prefixes] (in order and ignoring case),
+ * returns a copy of this string with the prefixes removed. Otherwise, returns this string.
+ */
+fun String.removePrefixesIgnoreCase(vararg prefixes: String): String {
```

Comment:
For later: Might make sense to move this into Android components (e.g. the `support-test` or a new `support-android-test` component.)

Reply:
thank you @csadilek! we'll work on getting this into android-components in a later pr

Reply:
Yeah, thought the same about `SessionLoadedIdlingResource` which is in all our apps now. :)

Also see https://github.com/mozilla-mobile/android-components/pull/1641 which will add some androidTest helpers too and pave the way for doing that.